test/commands/platforms/create.test.ts

```diff
@@ -73,10 +82,61 @@ describe('platform:create', function () {
     const kubernetes_configuration_fake = sinon.fake.returns({ name: 'new_k8s_platform', type: 'KUBERNETES' });
     sinon.replace(KubernetesPlatformUtils, 'configureKubernetesPlatform', kubernetes_configuration_fake);
 
-    await PlatformCreate.run(['platform-name', '-a', 'test-account-name', '-t', 'kubernetes', '--auto-approve']);
+    await PlatformCreate.run(['platform-name', '-a', 'test-account-name', '-t', 'kubernetes']);
     expect(create_platform_spy.calledOnce).true;
     expect(post_to_api_spy.calledOnce).true;
     expect(kubernetes_configuration_fake.calledOnce).true;
     expect(create_platform_applications_spy.calledOnce).true;
   });
+
+  it('Creates an Agent platform with input', async () => {
```

Comment:
I know the rest of this file is written this way, but this should use the newer format of writing tests rather than the old format. Personally I think it's way more readable

ex. https://github.com/architect-team/architect-cli/blob/rc/test/commands/dev.test.ts#L676